160718 yg report work

yguimaraes opened the pull request.

yguimaraes committed:
Before Ciclo de Vida

yguimaraes committed:
Before Ciclo de Vida

yguimaraes committed:
Fix all errors on botnets chapter after VE

yguimaraes committed:
Fix all errors on botnets chapter after VE

yguimaraes committed:
Report on new format compiling

yguimaraes committed:
Report on new format compiling

yguimaraes committed:
Update dados_pfc

yguimaraes committed:
Update dados_pfc

yguimaraes committed:
Add resumo and abstract

yguimaraes committed:
Add resumo and abstract

yguimaraes committed:
Fix citations

yguimaraes committed:
Fix citations

yguimaraes committed:
Fix citations

yguimaraes committed:
Fix citations

jonasrla commented on report/botnets.tex:
```diff
@@ -1,23 +1,33 @@
+As botnets apresentam alguns elementos estruturais tipicamente envolvidos, que estão presentes independente do protocolo ou arquitetura utilizada. A Figura \ref{fig:typical_elements} mostra a estrutura desses elementos e como eles se relacionam em uma botnet. Segue uma descrição para cada componente:
+\begin{itemize}  
+\item Bots: São \textit{malwares} instalados nos computadores das vítimas que podem realizar as ações maliciosas que o \textit{botmaster} envia através do canal de comando e controle (C\&C). Geralmente, o \textit{malware} é inicializado quando o hospedeiro inicializa a máquina, porém isso pode ser configurado pelo \textit{botmaster} para dificultar a detecção da atividade maliciosa.
+\item Hospedeiros (Zumbis): São as máquinas em que o bot foi instalado, ou seja, infectadas \citep{puri2003bots}.
```
O termo zumbi está na literatura?

jonasrla commented on report/botnets.tex:
```diff
@@ -1,23 +1,33 @@
 
-O mecanismo que estamos desenvolvendo é da forma reativa, já que o algoritmo encontrará padrões em botnets que já estão atuando. Porém, uma característica desejável para um detector reativo é a detecção em tempo real, com o objetivo de minimizar os danos causados e o tempo de reação do \textit{botmaster}. Porém, essa característica é um desafio, devido ao grande número de dados que devem ser trat ...
+O mecanismo que estamos desenvolvendo é da forma preventiva, já que o algoritmo busca encontrar padrões e identificar possíveis máquinas infectadas por botnets. Tudo isso na fase em que o \textit{botmaster} ainda está infectando máquinas com o bot para a botnet. Ou seja, buscamos detectar as máquinas infectadas antes do ataque, como um ataque de negação de serviço, em si ser efetivado
+. Uma característica desejável para um detector é a detecção em tempo real, com o objetivo de minimizar os danos causados e o tempo de reação do \textit{botmaster}. Porém, obter essa característica é um desafio, devido ao grande número de dados que devem ser tratados e analisados. Dessa forma, nosso projeto não fará detecção em tempo real, mas tentará se aproximar disso, utilizando a detecção dos ...
```
Esse ponto no começo da linha tá estranho, esse enter foi acidental?

jonasrla commented on report/botnets.tex:
```diff
@@ -45,40 +55,44 @@ \section{Ciclo de Vida das Botnets}
 \end{figure}
 
-Na primeira fase, chamada de injeção inicial, o atacante procura vulnerabilidades na máquina do futuro hospedeiro para explorá-las e infecta-lo com o malware, tornando-se um bot em potencial, isso pode ocorrer, por exemplo, através de um download indesejado ou através de um anexo em um e-mail. Após a infecção ser bem sucedida, ocorre a injeção secundária: o host infectado, através do malware inic ...
+Na primeira fase, chamada de injeção inicial, o atacante procura vulnerabilidades na máquina do futuro hospedeiro para explorá-las e infectá-lo com o \textit{malware}, tornando-se um bot em potencial, isso pode ocorrer, por exemplo, através de um \textit{download} indesejado ou através de um anexo em um e-mail. Após a infecção ser bem sucedida, ocorre a injeção secundária: o host infectado, atrav ...
```
bot não é estrangeirismo também?

jonasrla committed:
Ajuste de termo

feature -> característica
clustering -> agrupamento
aprendizagem de máquina -> aprendizado de máquina

jonasrla committed:
Limpa título da sessão

jonasrla committed:
writes Dado Bruto section

jonasrla committed:
writes Banco de Dados section

jonasrla committed:
fix new line accident

yguimaraes committed:
Fix conflicts

yguimaraes committed:
Remove zumbis

yguimaraes committed:
Remove image

yguimaraes committed:
Now compiles

yguimaraes committed:
Remove files on git ignore
